packages/boot/test/fast-usdc/snapshots/fast-usdc.test.ts.md

```diff
@@ -0,0 +1,19 @@
+# Snapshot report for `test/fast-usdc/fast-usdc.test.ts`
+
+The actual snapshot is saved in `fast-usdc.test.ts.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## writes feed policy to vstorage
+
+> Under "published", the "fastUsdc.feedPolicy" node is delegated to the general and chain-specific policies for the Fast USDC feed.
+> The example below illustrates the schema of the data published there.
+> 
+> See also board marshalling conventions (_to appear_).
+
+    [
+      [
+        'published.fastUsdc.feedPolicy',
+        '{"blockHeight":"0","values":["{\\"chainPolicies\\":{\\"Arbitrum\\":{\\"cctpTokenMessengerAddress\\":\\"0x19330d10D9Cc8751218eaf51E8885D058642E08A\\",\\"chainId\\":42161,\\"confirmations\\":1,\\"nobleContractAddress\\":\\"0x19330d10D9Cc8751218eaf51E8885D058642E08A\\"}},\\"nobleAgoricChannelId\\":\\"channel-21\\",\\"nobleDomainId\\":4}"]}',
```

Comment:
oops. `documentStorageSchema` doesn't seem to grok the plain JSON style.

Oh well. Here's hoping it will in due course.